packages/v4/src/content/accessibility/progress-stepper/progress-stepper.md

```diff
@@ -0,0 +1,20 @@
+---
+id: Progress stepper
+section: components
+---
+
+A **progress stepper** is used to provide information to the user about the state of their current progress on a multistep interaction.
+
+**Keyboard users** use standard keyboard navigation (**Tab** and **Shift + Tab**) to move to and between progress steps with popovers. Keyboard users should
+be able to open the popover on a progress step using **Enter** or **Space**, and close the popover using **Enter**, **Space**, or **Escape**.
+
+**Screen reader users** should be able to discern the label, status, and optional description of every progress step 
+via the `aria-label` prop. This should include all information being communicated by the progress step's icon, such as 
+the variant and the completed status, as well as whether the step has an associated popover they can trigger. Screen 
+reader users should also be able to access the contents of any popover associated with a step.
+
```

Comment:
So I think that we probably need to add an "Accessibility Application" section or whatever we're planning on calling it here. (I think we've called it different things on different a11y pages.) 
When we look at the progress stepper react demos, we're creating a live region that we're announcing the progress with. This should probably be explained in the a11y docs:
``` 
     <div className="pf-screen-reader" aria-live="polite">
          {steps[currentStep] && `On ${steps[currentStep].title}.`}
          {steps[currentStep - 1] && `${steps[currentStep - 1].title} was successful.`}
     </div>
     <ProgressStepper>
          {steps.map((step, index) => {
            let variant = 'pending';
            let ariaLabel = 'pending step';
            if (index < currentStep) {
              variant = 'success';
              ariaLabel = 'completed step, step with success';
            } else if (index === currentStep) {
              variant = 'info';
              ariaLabel = 'current step';
            } 
 ```